documentation/src/docs/reader-group-design.md

```diff
@@ -79,12 +78,16 @@ There are no races between multiple Readers coming online concurrently because o
 
 There is no ambiguity as to who the owner is, because it is stored in the shared state. There is no risk of a segment being ignored because every Reader can see the available segments by looking at the shared state and claim them.
 
-## Reader going offline.
-1. When a Reader dies, the `ReaderOffline()` method will be invoked either by the Reader itself in a graceful shutdown (internally to the close method) or via a "liveness detector". In either case the Reader's last position is written to the state.
-1. Other Readers will see this when they update their local state.
+## Reader going offline
+1. When a Reader dies, the `ReaderOffline()` method will be invoked either by the Reader itself in a graceful shutdown (internally to the close method) or via a "liveness detector". In either case the Reader's last position is written to the state `readerOffline(readerId, position) api`.
```

Comment:
readerOffline(readerId, position) method starts with lower case, right? 
We could replace `ReaderOffline()` with the actual api.
The second `readerOffline` reference seemed redundant.

Reply:
Accepted suggestion